src/Bridge/Doctrine/Orm/ItemDataProvider.php

```diff
@@ -74,6 +75,12 @@ public function getItem(string $resourceClass, $id, string $operationName = null
 
         foreach ($this->itemExtensions as $extension) {
             $extension->applyToItem($queryBuilder, $queryNameGenerator, $resourceClass, $identifiers, $operationName);
+
+            if ($extension instanceof QueryResultItemExtensionInterface) {
```

Comment:
Thought about this while I was copying files, wonder why it wasn't like this in the beginning. +1 ofc